### ISSUE TITLE
Issues on lmpm training

### ISSUE THREAD

**Issue by golddohyun** [henghuiding/MeViS#12]
I am trying to reproduce the baseline lmpm model, but I am getting results that are about 5 points lower on the J&F metric compared to what you reported in your paper.

Before training, I was instructed to download the backbone weights (model_final_86143f.pkl). Could you please clarify where in the code these weights are being used? Are these backbone weights actually used during the training process?

If they are used, where should I specify the path to the backbone weights in the training script?

**Comment by heshuting555** [comment]
Thank you for your interest in our work!

Backbone weights is used in the training process.

step1: python tools/process_ckpt.py
step2: python train_net_lmpm.py \
    --config-file configs/lmpm_SWIN_bs8.yaml \
    --num-gpus 8 --dist-url auto \
    MODEL.WEIGHTS model_final_86143f.pkl

**Comment by henghuiding** [comment]
Please kindly submit your **val set** results to [**CodaLab online evaluation**](https://codalab.lisn.upsaclay.fr/competitions/15094). For offline validation on you local machine, please use **Val<sup>u</sup> set**

The **test set** is used for challenges, such as the [Large-scale Video Object Segmentation](https://lsvos.github.io/) which has been ended last week. Please stay tuned if you are interested in the future challenges.

**Comment by golddohyun** [comment]
Ah okay thanks for the reply!
